### PR TITLE
142 remove dead code

### LINKED ISSUE #142: Remove dead code

### PR DESCRIPTION
**Pull request by gh-23378**

### CODE REVIEW

File: src/store/embalm/actions.ts [hunk 1]
```diff
@@ -145,15 +143,6 @@ export function toggleStep(step: Step): EmbalmActions {
...
 }
 
-export function setDiggingFees(diggingFees: string): EmbalmActions {
-  return {
-    type: ActionType.SetDiggingFees,
-    payload: {
-      diggingFees,
-    },
-  };
-}
```
Comment (sethhrbek) [inline src/store/embalm/actions.ts:155]: Should the other code related to this get removed as well -- ActionType + related reducer code?

File: src/store/embalm/actions.ts [hunk 2]
```diff
@@ -374,11 +354,4 @@ export function disableSteps(): EmbalmActions {
   };
 }
 
-export function enableSteps(): EmbalmActions {
-  return {
-    type: ActionType.EnableSteps,
-    payload: {},
-  };
-}
```
Comment (sethhrbek) [inline src/store/embalm/actions.ts]: I think this should stay -- the code using this function is temporarily commented out, but we will need re-enable the steps here (unless we change the "disable steps" function to "toggle steps")

File: src/hooks/useSubmitTransaction.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ export function useSubmitTransaction(
       });
     },
     onError(error) {
-      console.log('Transaction failed with args\n:', JSON.stringify(contractConfig.args));
+      console.log('createSarcophagus failed with args\n:', JSON.stringify(contractConfig.args));
```
Comment (DarksightKellar) [inline src/hooks/useSubmitTransaction.ts]: This isn't specifically a createSarcophagus transaction, although it was only being used for it, until a couple merges ago

**Review by DarksightKellar — CHANGES_REQUESTED** [review]

File: src/hooks/useSubmitTransaction.ts (hunk 3 above)
Comment (gh-23378) [reply src/hooks/useSubmitTransaction.ts]: good call! fixing

**Pull request merged**